appserver/web/war-util/src/main/java/org/glassfish/web/loader/WebappClassLoader.java

```diff
@@ -341,6 +333,20 @@ public void setAntiJARLocking(boolean enable) {
     }
 
 
+    /**
+     * Enables checks for the package definition permissions.
+     *
+     * @param enable
+     */
+    public void setPackageDefinitionSecurityEnabled(boolean enable) {
+        if (enable && SECURITY_MANAGER == null) {
```

Comment:
Note that eventually we'd have to remove all uses of the security manager

Reply:
Correct, but while it is there, it should work :-)